helm/istio/charts/kiali/templates/kiali-cr.yaml

```diff
@@ -22,7 +22,7 @@ spec:
 {{- if .Values.global.imagePullSecrets }}
     image_pull_secrets: 
 {{- range .Values.global.imagePullSecrets }}
-    - name: {{ . }}
+    - {{ . }}
```

Comment:
Note that Kubernetes style (https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/) is to use name: foo, but Kiali adds this internally when it applies the configuration rather than reading it out of the CR

Reply:
cc @jmazzitelli

Reply:
I think it is because of the kiali_cr is an ansible based operator and following helm/Kubernertes patterns.